java/server/src/org/openqa/selenium/grid/data/DistributorStatus.java

```diff
@@ -117,6 +117,10 @@ public boolean isUp() {
       return up;
     }
 
+    public boolean isDocker() {
```

Comment:
Prefer a human-readable string rather than querying specific technologies. How would I indicate a session is running on BrowserStack? Or some custom thing?